api.md

```diff
@@ -1034,6 +1035,43 @@ Create new databases `dist_database` on data nodes, which requires to set `trans
 CALL distributed_exec('CREATE DATABASE dist_database', transactional => FALSE);
 ```
 
+---
+## create_distributed_restore_point() [](create_distributed_restore_point)
+
+Create a named point for performing restore across a distributed database.
+
+This function is an analogue of `pg_create_restore_point()` designed to work with a distributed database
+and taking into account synchronization of the remote transactions.
+
+#### Required Arguments [](create_distributed_restore_point-required-arguments)
+
+|Name|Description|
+|---|---|
+| `name` | The restore point name. |
+
+#### Returns
+
+| Column          | Description                               |
+|-----------------|-------------------------------------------|
+| `node_name`     | Node name (or `NULL` for access node).    |
+| `node_type`     | Node type (`access_node` or `data_node`). |
+| `restore_point` | Restore point LSN.                        |
+
+#### Sample Usage [](create_distributed_restore_point-examples)
+
+Create the restore point `pitr` across three data nodes and access node:
+
+```sql
+SELECT * FROM create_distributed_restore_point('pitr');
+ node_name |  node_type  | restore_point 
+-----------+-------------+---------------
+           | access_node | 0/3694A30
+ dn1       | data_node   | 0/3694A98
+ dn2       | data_node   | 0/3694B00
+ dn3       | data_node   | 0/3694B68
+(4 rows)
+```
+
```

Comment:
It is good to describe all cases when an error will happen.

Reply:
I've updated the description with requirements, not really sure what other errors are expected to be described here

Reply:
According to the implementation there are many cases when error will be returned. See https://github.com/timescale/timescaledb/pull/2843/files#diff-cf55e2617d97e45d62909cc52c6382e2ae31e0cb57e40e246638c73036f6d057R77-R114